tests/test_vmec.py

```diff
@@ -483,12 +483,16 @@ def test_vmec_save_1(VMEC_save):
     )
     np.testing.assert_allclose(vmec.variables["chipf"][:], desc.variables["chipf"][:])
     np.testing.assert_allclose(
-        vmec.variables["Rmajor_p"][:], desc.variables["Rmajor_p"][:]
+        vmec.variables["Rmajor_p"][:], desc.variables["Rmajor_p"][:], rtol=3e-4
     )
     np.testing.assert_allclose(
-        vmec.variables["Aminor_p"][:], desc.variables["Aminor_p"][:]
+        vmec.variables["Aminor_p"][:],
+        desc.variables["Aminor_p"][:],
+        rtol=2e-4,
```

Comment:
shouldn't the new method of computing things be even more accurate?

Reply:
In the limit toward high resolution yes. Looks like at the resolution used in this [test](https://github.com/PlasmaControl/DESC/blob/4b21abcc54f898a07d478edfc263a5f9cb0d6cd5/tests/conftest.py#L328), the new method agrees less with vmec.

I could update my sympy script to compute the analytic result of these finite quadrature sampled on this grid for the SOLOVEV's boundary. This would prove whether the new method is less accurate at this resolution for these sample points.

I haven't done that to confirm. However, I had done similar for other surfaces [e.g. here](https://github.com/PlasmaControl/DESC/pull/807#issuecomment-1881858032), and the numerical quadrature agreed exactly with the analytic quadrature (both of which converge to the true integral).

Reply:
i don't think my previous comment is relevant